index.html

```diff
@@ -1,544 +1,627 @@
+            </fieldset>
+
+            <fieldset class="field">
+              <label class="label" for="subject">Subject</label>
+              <div class="control">
+                <div class="select">
+                  <select id="subject" name="subject">
+                    <option>Select dropdown</option>
+                    <option>With options</option>
+                  </select>
                 </div>
               </div>
+            </fieldset>
+
+            <fieldset class="field">
+              <label class="label" for="message">Message</label>
+              <div class="control">
+                <textarea id="message" class="textarea" placeholder="Enter your message"
+                  name="message"></textarea>
+              </div>
+            </fieldset>
+
+            <fieldset class="field">
+              <label class="checkbox">
+                <input id="terms" type="checkbox" name="terms">
+                I agree to the <a href="#">terms and conditions</a>
+              </label>
+            </fieldset>
+
+            <fieldset class="field">
+              <legend class="label">Subscribe to our newsletters?</legend>
+              <div class="control">
+                <label class="radio">
+                  <input id="yes" type="radio" name="yesno" value="yes">
+                  Yes
+                </label>
+                <label class="radio">
+                  <input id="no" type="radio" name="yesno" value="no">
+                  No
+                </label>
+              </div>
+            </fieldset>
+
+            <fieldset class="field is-grouped">
+              <div class="control">
+                <button type="submit" class="button is-link">Submit</button>
+              </div>
+              <div class="control">
+                <button type="button" class="button is-text">Cancel</button>
+              </div>
+            </fieldset>
+          </form>
+        </div>
+      </section>
+
+      <section class="section">
+        <div class="container">
+          <div class="section-heading has-text-centered">
+            <h2 class="title is-2">Forum</h2>
+            <h3 class="subtitle is-5">Discuss any topic with no leaving this
+              page</h3>
+          </div>
+          <div class="columns">
+            <div class="column is-3">
+              <a href="/todo"
+                class="button is-primary is-block is-alt is-large">New Post</a>
             </div>
+            <div class="column is-9">
+              <div class="box content">
+                <article class="post">
+                  <h1>How do you center a button in a box?</h1>
+                  <div class="media">
+                    <div class="media-left">
+                      <p class="image is-32x32">
+                        <img
+                          src="https://source.unsplash.com/random/64x64/?sig=300"
+                          alt>
```

Comment:
This image won't be read by a screenreader with alt missing. You can add something like `Jane Smith avatar`